py/selenium/webdriver/common/utils.py

```diff
@@ -102,10 +102,16 @@ def is_connectable(port, host="localhost"):
      - port - The port to connect.
     """
     socket_ = None
+
+    if sys.version_info > (3, 4):
+        expected_exceptions = (socket.error, ConnectionResetError)
+    else:
+        expected_exceptions = (socket.error,)
+
```

Comment:
You can throw this at the top with:

https://github.com/SeleniumHQ/selenium/blob/0d40dcd64295047d8669e688927a414d399e712e/py/selenium/webdriver/common/utils.py#L24-L28

Reply:
Thanks for the review! Sure -- done.